sdk/python/testvectors/nem/mosaic_definition.py

```diff
@@ -0,0 +1,173 @@
+			'mosaic_definition': {
+				'owner_public_key': '69AF8763FEECEF35E0AEF44A202EFAC4936532C202784CF85CBCD16BFC45F119',
+				'id': {
+					'namespace_id': {'name': b'state.sponsored'},
+					'name': b'terrorism',
+				},
+				'description': b'precious vouchers',
+				'properties': [
+					{'property_': {'name': b'divisibility', 'value': b'2'}},
+					{'property_': {'name': b'initialSupply', 'value': b'1000'}},
+					{'property_': {'name': b'supplyMutable', 'value': b'true'}},
+					{'property_': {'name': b'transferable', 'value': b'true'}}
+				],
+				'levy_size': 0x0,
+			},
+			'rental_fee_sink': 'TARPSQFPJL6A2ORAQJ46GOZUDPNYVQJKGVT2NMG7',
+			'rental_fee': 0x0000000028697580
+		},
+
+		'MosaicDefinitionTransaction_4': {
+			'signature': nc.Signature(
+				'FDE6262170C31B7AFD736B2BD4871FAEC176F02C6626A1F67A3B295F129145E7'
+				'89517C483FEA9E72FA08B808ED3A528B7FD514186E80FBF4E868CA7A84CE4309'),
+			'type': 'mosaic_definition_transaction',
+			'version': 0x1,
+			'network': 'testnet',
+			'timestamp': 0x0001E240,
+			'signer_public_key': 'B5B93DDE2D05D21D0A14E6F60DB33E983B88A99FB203916D408AF5749A396960',
+			'fee': 0x00000000066FF300,
+			'mosaic_definition': {
+				'owner_public_key': 'B5B93DDE2D05D21D0A14E6F60DB33E983B88A99FB203916D408AF5749A396960',
+				'id': {
+					'namespace_id': {'name': b'mass.surveillance'},
+					'name': b'electronic',
+				},
+				'description': b'precious vouchers',
+				'properties': [
+					{'property_': {'name': b'divisibility', 'value': b'2'}},
+					{'property_': {'name': b'initialSupply', 'value': b'1000'}},
+					{'property_': {'name': b'supplyMutable', 'value': b'true'}},
+					{'property_': {'name': b'transferable', 'value': b'true'}}
+				],
+				'levy_size': 0x54,  # note: needs to be here, as it is not derived now...
+				'levy': {
+					'transfer_fee_type': 'absolute',
+					'recipient_address': 'TCUTCNM64Y6Q4VB4OTEHBQT2ZKUY3CUYRVHCTIZ3',
+
+					'mosaic_id': {
+						'namespace_id': {'name': b'reptilians'},
+						'name': b'united',
+					},
+					'fee': 0x00000000037706A6,
+				}
+			},
+			'rental_fee_sink': 'TD3M6P3CENCDVIC2GRJLQPLQXBX6MOB6FBQZOXM2',
+			'rental_fee': 0x0000000028697580,
+		},
+
+		'MosaicDefinitionTransaction_5': {
+			'signature': nc.Signature(
+				'24056CEC2CFCC0C834336BB19701B91FF9129E0F1EF440441D3FB400D69F285F'
+				'B3BF5E8C6139D181ADD15D2918A623DBF130D14A3047A26C38DA72E9BD62AE08'),
+			'type': 'mosaic_definition_transaction',
+			'version': 0x1,
+			'network': 'testnet',
+			'timestamp': 0x0001E240,
+			'signer_public_key': 'D7F44870D24E7626DB24591452A2F7ECF6650B0D41D3BCB4FB4BA11B063B80AC',
+			'fee': 0x00000000066FF300,
+			'mosaic_definition': {
+				'owner_public_key': 'D7F44870D24E7626DB24591452A2F7ECF6650B0D41D3BCB4FB4BA11B063B80AC',
+				'id': {
+					'namespace_id': {'name': b'mass.surveillance'},
+					'name': b'electronic',
+				},
+
+				'description': b'precious vouchers',
+				'properties': [
+					{'property_': {'name': b'divisibility', 'value': b'2'}},
```

Comment:
i still wish we had better solution to foo_

Reply:
we could add `REWRITE_HINTS` akin to `TYPE_HINTS`
map like: "property" -> "property_"